README.md

```diff
@@ -22,12 +33,105 @@ It supports TensorFlow, PyTorch, MXNet, and XGBoost on Python 3.6+.
+
+   - Introducing experimental support for TF 2.x training scripts using GradientTape -
+   With this update, weights, bias, loss, metrics, and gradients are captured by SageMaker Debugger.
+   GradientTape in TF 2.x captures these tensors from custom training jobs. An example of GradientTape implementation to a custom ResNet training script using TensorFlow's Keras interface is provided at [tf_keras_gradienttape.py](https://github.com/awslabs/sagemaker-debugger/blob/master/examples/tensorflow2/scripts/tf_keras_gradienttape.py).
+   GradientTape does not work with Zero Script Change experience at this time.
+
+        *Note*: Training scripts using GradientTape for higher-order gradients or multiple tapes are not supported. Distributed training scripts that use GradientTape are not supported at this time.
+
+   - Support `SyncOnReadVariable` in mirrored strategy - Fixes a bug that occurred because the `SyncOnRead` distributed variable was not supported with `smdebug`. Also enables the use of `smdebug` with training scripts using TF 2.x MirroredStrategy with the `fit()` API.
+
+   - Turn off hook and write only from one worker for unsupported distributed training techniques – Fixes a crash when distributed training in PyTorch framework is implemented using generic multiprocessing library, which is not a method supported by `smdebug`. This fix handles this case and ensures that tensors are saved.
+
+   - Bug fix: Pytorch: Register only if tensors require gradients – Users were observing a crash when training with pre-trained embeddings which does not need gradient updates. This fix checks if a gradient update is required and registers a backward hook only in those cases.
+
+### Setting up SageMaker Debugger
+
+`smdebug` library runs on Python 3.x. Install `smdebug` through:
+
+```
+pip install smdebug
+```
+
+#### Zero Script Change
+
+You can use your own training script while using [AWS Deep Learning Containers (DLC)](https://aws.amazon.com/machine-learning/containers/) in TensorFlow, PyTorch, MXNet and XGBoost frameworks. The AWS DLCs enable you to use Debugger with no changes to your training script by automatically adding SageMaker Debugger's `Hook`.
+The following table shows currently supported versions of the four frameworks for Zero Script Change experience.
+
+| Framework | Version |
+| --- | --- |
+| [TensorFlow](tensorflow.md) | 1.15, 2.1 |
+| [MXNet](mxnet.md) | 1.6 |
+| [PyTorch](pytorch.md) | 1.3, 1.4 |
+| [XGBoost](xgboost.md) | >=0.90-2 [As Built-in algorithm](xgboost.md#use-xgboost-as-a-built-in-algorithm)|
+
+
+#### Bring your own training container
+
+| Framework | Versions |
+| --- | --- |
+| [TensorFlow](tensorflow.md) | 1.14. 1.15, 2.0.1, 2.1.0 |
+| Keras (with TensorFlow backend) | 2.3 |
+| [MXNet](mxnet.md) | 1.4, 1.5, 1.6 |
+| [PyTorch](pytorch.md) | 1.2, 1.3, 1.4 |
+| [XGBoost](xgboost.md) | [As Framework](xgboost.md#use-xgboost-as-a-framework) |
+
```

Comment:
example, or steps defining how they can use debugger with BYOC

Reply:
It would be nice if you can provide a code or a page for it.

Reply:
Added on of the new BYOC snippet of code using TF 2.x GradientTape from tensorflow.md